server/src/app/GaWorker.ts

```diff
@@ -113,4 +113,58 @@ export class GaWorker {
 
     return startSet.map((s: string) => new Note(s));
   }
+
+  // Function to assign weights to possible notes for mutation
```

Comment:
Can you document the expected behaviour of this function? Ie explain the calculations / formulas that go into determining the weight given to a note mutation based on whether it was in octave or a step?

```
/**
Something like this function documentation here
*/
```